chord_metadata_service/chord/views_ingest.py

```diff
@@ -222,7 +222,7 @@ def ingest_phenopacket(phenopacket_data, table_id):
             term=disease["term"],
             disease_stage=disease.get("disease_stage", []),
             tnm_finding=disease.get("tnm_finding", []),
-            **_query_and_check_nulls(disease, "onset")
+            onset=disease.get("onset", None)
```

Comment:
`_query_and_check_nulls` is meant to do the `=None` check without causing issues

Reply:
yeah, I see here, I don't understand why then it didn't save the "onset" value in db.